common/common.scss

```
@@ -581,3 +581,7 @@ html body #main-outlet {
 .topic-list .main-link.focused {
   box-shadow: none;
 }
+
+.title-voting {
+  padding-left: 14px
```

```suggestion
  padding-left: 14px;
```

Probably why linter is whining.